robocorp-code/vscode-client/src/inspector.ts

```diff
@@ -9,17 +9,34 @@ import { OUTPUT_CHANNEL } from "./channel";
 import { ChildProcess } from "child_process";
 import { feedback } from "./rcc";
 
-let _openingInspector: boolean = false;
-let _startingRootWindowNotified: boolean = false;
+export enum InspectorType {
+    Browser = "browser",
+    Windows = "windows",
+    Image = "image",
+    WebRecorder = "web-recorder",
+}
+
+export type InspectorTypes = `${InspectorType}`;
+
+export const DEFAULT_INSPECTOR_VALUE = {
```

Comment:
I don't really see a reason for the `DEFAULT_INSPECTOR_VALUE` (since it'll always be `false` for everything)